### PR TITLE
DOCS-12295 Mailchimp Marketing 1.0

### PR DESCRIPTION
**Pull request by sathya0**

### CODE REVIEW

File: mailchimp-marketing/1.0/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+* Credentials to access the Mailchimp Marketing target resource
+* Anypoint Platform
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Some common use cases for Mailchimp Marketing Connector include:
+
+*
```
Comment (adrian-sv) [inline mailchimp-marketing/1.0/modules/ROOT/pages/index.adoc]: * Creating and sending campaigns
* Adding, updating, and removing subscribers from lists
* Monitoring subscribers/unsubscribers counts and campaign email open rates

**Review by adrian-sv — DISMISSED** [review]

**Review by sorinmare — DISMISSED** [review]
LGTM

File: mailchimp-marketing/1.0/modules/ROOT/pages/mailchimp-marketing-connector-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,1412 @@
...
+** NEVER | How the connector streams request content to the remote service:
+
+* `AUTO` (default)
++
+Connector automatically uses the best streaming strategy based on the request content.
+
+* `ALWAYS`
++
+Connector always streams the request content.
+
```
Comment (dhirsch1) [inline mailchimp-marketing/1.0/modules/ROOT/pages/mailchimp-marketing-connector-reference.adoc:43]: Most of the rendered list items show "+" in the text. It looks like the auto-generation process left out "+" in some places so the lists were rendered incorrectly.

File: mailchimp-marketing/1.0/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Anypoint Connector for Mailchimp Marketing (Mailchimp Marketing Connector) enables you to integrate your applications with the Mailchimp Marketing data and functionality. This connector exposes operations provided by the Mailchimp Marketing API.
+
+For information about compatibility and fixed issues, refer to the Mailchimp Marketing Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Mailchimp Marketing Connector’s API
```
Comment (dhirsch1) [inline mailchimp-marketing/1.0/modules/ROOT/pages/index.adoc]: This should be "The Mailchimp Marketing API." The template wording is unclear.

**Review by dhirsch1 — DISMISSED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**